### PR TITLE
Auth token working, dynamic scroll incomplete

### PR DESCRIPTION
**Pull request by harshLFC**

### CODE REVIEW

File: lib/services/auth.dart [hunk 1]
```diff
@@ -48,13 +45,30 @@ class AuthService {
...
+    try {
+      final test = await FirebaseAuth.instance.currentUser();
+      final idToken = await test.getIdToken();
+      return idToken;
+    } catch (e) {
+      print(e.toString());
+      return null;
+    }
+  }
+
```
Comment (sanketdayama) [inline lib/services/auth.dart:71]: The or unusable code and print statements can be removed

File: lib/screens/prediction.dart [hunk 2]
```diff
@@ -1,258 +1,259 @@
...
-      'leeds',
-      'fulham',
-      'brighton',
-      'southampton'
-    ];
+//   @override
+//   Widget build(BuildContext context) {
+//     final homeTeams = [
+//       'arsenal',
+//       'chelsea',
```
Comment (sanketdayama) [inline lib/screens/prediction.dart:151]: These comments too can be removed

**Review by sanketdayama — CHANGES_REQUESTED** [review]
Hi Harsh, I have written some comments for Auth, for logic I just went through once maybe once I do the forgot password I'll get a better sense of it

**Comment by harshLFC** [comment]
Sure

Regards
Harsh Phoujdar

On 23-Jan-2021, at 6:25 PM, sanketdayama <notifications@github.com> wrote:

﻿

@sanketdayama requested changes on this pull request.

Hi Harsh, I have written some comments for Auth, for logic I just went through once maybe once I do the forgot password I'll get a better sense of it

File: lib/services/auth.dart (hunk 1 above)
Comment (harshLFC) [reply lib/services/auth.dart:71]: Thanks, will do

File: lib/screens/prediction.dart (hunk 2 above)
Comment (harshLFC) [reply lib/screens/prediction.dart:151]: Ok

**Pull request closed without merging**